insert-toc/info.json

```diff
@@ -2,7 +2,7 @@
   "name": "Insert Table of Contents (TOC)",
   "identifier": "insert-toc",
   "script": "insert-toc.qml",
-  "authors": ["@flopp"],
+  "authors": ["@flopp", "@toindev"],
   "platforms": ["linux", "macos", "windows"],
   "version": "0.0.1",
```

Comment:
@toindev please also increase the version number, so the script can get an update